### PR TITLE
Update manifest_integrity params

### PR DESCRIPTION
**Pull request by ens-LCampbell**
Remove boolean operands for params converted to optional flag

### CODE REVIEW

File: src/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_loader_conf.pm [hunk 1]
```diff
@@ -358,7 +358,7 @@ sub pipeline_analyses {
       -module      => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
       -parameters  => {
         cmd => 'mkdir -p #log_path#; '
-             . 'manifest_check_integrity --brc_mode #brc4_mode# --ignore_final_stops #ignore_final_stops# --manifest_file #manifest# '
+             . 'manifest_check_integrity --brc_mode --ignore_final_stops --manifest_file #manifest# '
```
[the remark this answers is not in the document]
Comment (vsitnik) [reply src/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_loader_conf.pm:361]: add something like (below) before cmd:
```
'brc4_mode_param' => '#expr( #brc4_mode# ? "--brc_mode" : "")expr#',
    
```

[the remark this answers is not in the document]
Comment (MatBarba) [reply src/perl/Bio/EnsEMBL/Pipeline/PipeConfig/BRC4_genome_loader_conf.pm:361]: That's exactly what I'm doing right now to make the pipeline run for the new build.

**Review by MatBarba — APPROVED** [review]
I used this fix to run brc build

**Review by vsitnik — APPROVED** [review]
thanks!

**Pull request merged**